.travis.yml

```diff
@@ -31,4 +31,6 @@ install:
 
 script:
     - pip install virtualenv==1.10
+    - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install "mercurial>=3.3" ; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install python-hglib==1.5 ; fi
```

Comment:
(Of course that makes sense about `apt-get` ... :open_mouth: )  I will absolutely issue a PR when `python-hglib` is ported.  Thanks!